### ISSUE TITLE
Signed build fails with warning. Version - 2.15.0

### ISSUE THREAD

**Issue by AnujMathu07** [braintree/braintree_android#227]
### General information

* SDK/Library version: 2.15.0
* Environment: building issue
* Android Version and Device: NA
* Braintree dependencies: 
com.braintreepayments.api:braintree:2.15.0
### Issue description

While creating signed build using latest dependency 2.15.0, build fails with warning. Below are the warning that we are getting:
```
Warning:com.braintreepayments.api.DataCollector: can't find referenced class com.paypal.android.sdk.data.collector.PayPalDataCollector
Warning:com.paypal.android.sdk.onetouch.core.PayPalOneTouchCore: can't find referenced class com.paypal.android.sdk.data.collector.PayPalDataCollector
Warning:com.paypal.android.sdk.onetouch.core.fpti.FptiManager: can't find referenced class com.paypal.android.sdk.data.collector.InstallationIdentifier
Warning:com.paypal.android.sdk.onetouch.core.sdk.AppSwitchHelper: can't find referenced class com.paypal.android.sdk.data.collector.InstallationIdentifier
```

But if we use dependency version 2.14.0, build is successful without any warning.

Please Provide required proguard rule to be added when changing version to 2.15.0

**Comment by Singloo** [comment]
same issue

**Comment by hibob224** [comment]
On 2.15.0 our devs on windows machines are getting the following when trying to build

```
Illegal char <:> at index 1: :PayPalDataCollector\build\libs\META-INF\MANIFEST.MF 
```

**Comment by Epreuve** [comment]
Thanks for the reports folks.  This is likely due to a dependency update we made.  We're looking at it now and will update when we have a fix released.

**Comment by quinnjn** [comment]
We released 2.15.2 which should have solved compile issues for PayPalDataCollector. Let us know if there are still issues compiling this version!

**Comment by quinnjn** [comment]
Closing this issue, please comment if the issue still exists in 2.15.2
